src/style.css

```diff
@@ -0,0 +1,150 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+.search {
+  background: rgb(255, 255, 255);
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+
+  /* margin-top: 20px;
+  background-image: url(https://raw.githubusercontent.com/serhii-vorona/layout_search-bar-airbnb/ed01024bb96e6ee32a7fa83a40afdcdad24efbd4/src/images/Search.svg);
+  background-repeat: no-repeat;
+
+  background-size: auto 30%;
+  background-position: center left 10px;
+  padding-left: 40px;
+  font-family: Avenir, sans-serif;
+  color: #3d4e61; */
+}
+
+/* .search:hover {
+  background-color: rgb(255, 255, 255);
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+  margin-top: 20px;
+  background-image: url(https://raw.githubusercontent.com/serhii-vorona/layout_search-bar-airbnb/ed01024bb96e6ee32a7fa83a40afdcdad24efbd4/src/images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: auto 30%;
+  background-position: center left 10px;
+  padding-left: 40px;
+}
+
+.search:focus {
+  background: linear-gradient(180deg, rgb(255, 255, 255) 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  border-radius: 3px;
+  margin-top: 20px;
+  background-image: url(https://raw.githubusercontent.com/serhii-vorona/layout_search-bar-airbnb/ed01024bb96e6ee32a7fa83a40afdcdad24efbd4/src/images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: auto 30%;
+  background-position: center left 10px;
+  padding-left: 40px;
+  font-weight: 900;
+} */
+
+form[data-qa="big"] {
+  margin-top: 20px;
+}
+
+form[data-qa="small"] {
+  margin-top: 20px;
+}
+
+#bigSearch {
+  width: 100%;
+  height: 70px;
+  font-size: 16px;
+  background-image: url(https://raw.githubusercontent.com/serhii-vorona/layout_search-bar-airbnb/ed01024bb96e6ee32a7fa83a40afdcdad24efbd4/src/images/Search.svg);
+  background-repeat: no-repeat;
+
+  background-size: auto 28%;
+  background-position: center left 25px;
+  padding-left: 62px;
+  font-family: Avenir, sans-serif;
+  color: #3d4e61;
+}
+
+#smallSearch {
+  width: 100%;
+  height: 42px;
+  font-size: 14px;
+  background-image: url(https://raw.githubusercontent.com/serhii-vorona/layout_search-bar-airbnb/ed01024bb96e6ee32a7fa83a40afdcdad24efbd4/src/images/Search.svg);
+  background-repeat: no-repeat;
+
+  background-size: auto 28%;
+  background-position: center left 12px;
+  padding-left: 33px;
+  font-family: Avenir, sans-serif;
+  color: #3d4e61;
+}
+
+#bigSearch:hover {
+  width: 100%;
+  height: 70px;
+  font-size: 16px;
```

Comment:
You shouldn't repeat every style in every state, use default style for inputs and than add for :hover and :focus styles that was changed